index.js

```diff
@@ -246,6 +248,11 @@ class ServerlessS3Local {
       return eventHandlers;
   }
 
+  getResourceForBucket(bucketName){
+    const logicalResourceName = `S3Bucket${bucketName.charAt(0).toUpperCase()}${bucketName.substr(1)}`;
+    return !this.service.resources || this.service.resources.Resources[logicalResourceName];
```

Comment:
Hehe yeah - I rushed this change. good pickups.  I'll fix these this weekend

Reply:
Want me to add in a test runner? - I use often tape as its quite lightweight.

Reply:
Ugh bad idea - too much to mock..

Reply:
Thanks for your efforts.
I think it is difficult to write test code for serverless-s3-local.
Because it act as plugin of serverless.
But test code is exactly useful.
I will also try !! 

And I didn't know about tape.
So it is good information for me.